corehq/util/couch_doc_processor.py

```diff
@@ -367,3 +395,65 @@ def _processing_complete(self):
         ))
         if self.skipped:
             print(DOCS_SKIPPED_WARNING.format(self.skipped))
+
+
+class BulkDocProcessorEventHandler(PaginateViewEventHandler):
+
+    def __init__(self, processor):
+        self.processor = processor
```

Comment:
I guess the same would apply to the `ResumableDocsByTypeEventHandler`. I can add that in a follow up PR.